src/seedu/addressbook/AddressBook.java

```diff
@@ -242,9 +224,7 @@ private static void showResultToUser(String result) {
      * ====================================================================
      */
 
-    /**
-     * Echoes the user input back to the user.
-     */
```

Comment:
I think for function, should stick to the 3 lines comment. See Coding Standard: https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments